app/frontend/src/components/EnvironmentBanner.tsx

```diff
@@ -0,0 +1,43 @@
+import { makeStyles, useMediaQuery, useTheme } from "@material-ui/core";
+import React from "react";
+
+const useStyles = makeStyles((theme) => ({
+  banner: {
+    position: "fixed",
+    top: 2,
+    left: 2,
+    zIndex: 5000,
+    paddingTop: 2,
+    paddingBottom: 2,
+    paddingLeft: 5,
+    paddingRight: 5,
+    opacity: 0.8,
+    borderRadius: theme.shape.borderRadius,
+    backgroundColor: theme.palette.error.main,
+    color: theme.palette.common.white,
+    pointerEvents: "none",
+    whiteSpace: "nowrap",
+    overflow: "hidden",
+  },
+}));
+
+export function EnvironmentBanner() {
+  const theme = useTheme();
+  const classes = useStyles();
+  const isBelowSm = useMediaQuery(theme.breakpoints.down("sm"));
+
+  if (process.env.REACT_APP_COUCHERS_ENV === "prod") {
+    return <></>;
```

Comment:
```suggestion
    return null;
```
React convention is to return null when you don't want to anything